tinymist: 0.11.22 -> 0.11.28

GaetanLepage opened the pull request:
## Things done

Changelog: https://github.com/Myriad-Dreamin/tinymist/blob/refs/tags/v0.11.28/CHANGELOG.md

cc @lampros

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

Myriad-Dreamin commented:
Hey, did you note the stable/nightly release note described in [releases/tag/v0.11.28](https://github.com/Myriad-Dreamin/tinymist/releases/tag/v0.11.28)?

GaetanLepage committed:
tinymist: 0.11.27 -> 0.11.28

Diff: https://github.com/Myriad-Dreamin/tinymist/compare/refs/tags/v0.11.27...0.11.28

Changelog: https://github.com/Myriad-Dreamin/tinymist/blob/refs/tags/v0.11.28/CHANGELOG.md

GaetanLepage commented:
> Hey, did you note the stable/nightly release note described in [releases/tag/v0.11.28](https://github.com/Myriad-Dreamin/tinymist/releases/tag/v0.11.28)?

Yes, I forgot for a moment.
I have reverted to updating to 0.11.28 now :)

Do you know when the vscode extension will be updated ?

Myriad-Dreamin commented:
My bad..

https://github.com/Myriad-Dreamin/tinymist/blob/485adc77613d57160b63e298172f9f81a8770706/editors/vscode/package.json#L3

GaetanLepage commented:
> My bad..
> 
> https://github.com/Myriad-Dreamin/tinymist/blob/485adc77613d57160b63e298172f9f81a8770706/editors/vscode/package.json#L3

All good now, thanks :à

GaetanLepage commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 346629`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>tinymist</li>
    <li>vscode-extensions.myriad-dreamin.tinymist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>tinymist</li>
    <li>vscode-extensions.myriad-dreamin.tinymist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>tinymist</li>
    <li>vscode-extensions.myriad-dreamin.tinymist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>tinymist</li>
    <li>vscode-extensions.myriad-dreamin.tinymist</li>
  </ul>
</details>
